### PR TITLE
Building chaos kernel for release 2.0

### PR DESCRIPTION
**Pull request by STRRL**
### What problem does this PR solve?

> Problem Summary:

it's a part of https://github.com/chaos-mesh/chaos-mesh/issues/2483

- `chaos-kernel` is not built with GitHub Action 

### What's changed and how it works?

- add `chaos-kernel` to upload_image.yaml

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Chaos Dashboard`
- [ ] Need to **cheery-pick to release branches**

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

I manually tested it at https://github.com/STRRL/chaos-mesh/actions/runs/1434823231 and https://github.com/STRRL/chaos-mesh/actions/runs/1437744350

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### CODE REVIEW

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by iguoyr — APPROVED** [review]
LGTM

File: images/chaos-kernel/Dockerfile [hunk 1]
```diff
@@ -36,6 +36,6 @@ WORKDIR /
 
 RUN git clone https://github.com/chaos-mesh/bpfki
 WORKDIR /bpfki/build
-RUN cmake .. && make -j && mv bin/bpfki /usr/local/bin/ && mv examples/fail* /usr/local/bin/
+RUN cmake .. && make -j 16 && mv bin/bpfki /usr/local/bin/ && mv examples/fail* /usr/local/bin/
```
Comment (YangKeao) [inline images/chaos-kernel/Dockerfile:39]: Maybe 4 is enough, for github action (as it has only 2 cpus).

Comment (STRRL) [reply images/chaos-kernel/Dockerfile:39]: how about `make -j $(nproc)`

Comment (STRRL) [reply images/chaos-kernel/Dockerfile:39]: or `$[$(nproc) + 1]` or `$[$(nproc) * 2]`?

Comment (STRRL) [reply images/chaos-kernel/Dockerfile]: I noticed here is an arg called `MAKE_JOBS`, I would use this arg with default value `4`.

**Pull request merged**